src/pages/developers/local-environment.tsx

```diff
@@ -1,279 +1,48 @@
 import React, { useEffect, useState } from "react"
-import styled from "@emotion/styled"
+import {
+  Box,
+  Flex,
+  Heading,
+  Img,
+  ListItem,
+  Text,
+  UnorderedList,
+} from "@chakra-ui/react"
+
 import { GatsbyImage, IGatsbyImageData } from "gatsby-plugin-image"
 import { graphql, PageProps } from "gatsby"
 import { useIntl } from "react-intl"
 import { shuffle } from "lodash"
 
-// import ActionCard from "../../components/ActionCard"
-// import Callout from "../../components/Callout"
-// import Card from "../../components/Card"
-// import Link from "../../components/Link"
-// import ButtonLink from "../../components/ButtonLink"
 import Translation from "../../components/Translation"
 import PageMetadata from "../../components/PageMetadata"
 import ProductCard from "../../components/ProductCard"
 import {
   Content,
   CardGrid,
-  // Divider,
-  // Intro,
   Page,
-  // InfoBanner,
 } from "../../components/SharedStyledComponents"
```

Comment:
As a quick review, one thing we want to deprecate is the usage of `SharedStyledComponents`. We need to replace those components with Chakra components.

Reply:
I'm sorry, it's been resolved.

Reply:
Nice! no need to say sorry. Thank you for the great work!